packages/core/src/styled/Chips/Chips.tsx

```diff
@@ -0,0 +1,77 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import React, { FC, forwardRef } from 'react';
+import { usePropsOverwrites } from '@core/styles';
+import { QX_SIZE } from '@core/enums';
+import { getRightIcon } from '@core/src/styled/Chips/utils';
+import { CHIPS_VARIANT } from './constants';
+import { ChipsProps } from './types';
+import { CHIPS_CSS_VARS, useStyles } from './styles';
+
+export const Chips: FC<ChipsProps> = forwardRef<HTMLButtonElement, ChipsProps>((
+    initialProps,
+    ref,
+) => {
+    const { cn, props } = usePropsOverwrites('Chips', initialProps, CHIPS_CSS_VARS);
+    const {
+        styles: externalStyles,
+        children,
+        leftIcon,
+        variant = CHIPS_VARIANT.input,
+        active,
+        elevation,
+        disabled,
+        size = QX_SIZE.base,
+        disableFocus,
+        cssVars,
+        rightIcon,
+        hidden,
+        ...restProps
+    } = props;
+
+    const params = {
+        variant,
+        size,
+        active: Boolean(active),
+        elevation: Boolean(elevation),
+        disabled: Boolean(disabled),
+        disableFocus: Boolean(disableFocus),
+        leftIcon: Boolean(leftIcon),
+        rightIcon: Boolean(rightIcon && getRightIcon(variant)),
+    };
+
+    const styles = useStyles({ ...params, cssVars, styles: externalStyles });
+
+    const RightIcon = getRightIcon(variant)?.[size];
+
+    if (hidden) { return null; }
+
+    return (
+        <button
+            ref={ref}
+            type="button"
+            className={cn('root', params)}
+            css={styles.root}
+            disabled={disabled}
+            {...restProps}
+        >
+            {leftIcon && (
+                <div
+                    className={cn('leftIcon', params)}
```

Comment:
`params` лучше передавать в классы рута, иначе могут быть логические несостыковки.

```suggestion
                    className={cn('leftIcon')}
```

Reply:
уже был такой коммент, исправлено